flink-contrib/docker-flink/docker-compose.yml

```diff
@@ -20,16 +20,20 @@ version: "2"
 services:
   jobmanager:
     image: flink
+    container_name: "jobmanager"
+    expose:
+      - "6123"
     ports:
       - "48081:8081"
     command: jobmanager
-    volumes:
-      - /opt/flink/conf
 
   taskmanager:
     image: flink
+    expose:
+      - "6121"
+      - "6122"
     depends_on:
       - jobmanager
     command: taskmanager
-    volumes_from:
-      - jobmanager:ro
+    links:
```

Comment:
`links` are now a legacy feature of Docker 1.9.0 but probably fine to stick with it for now.